build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql

```diff
@@ -212,3 +212,9 @@ ALTER TABLE tadetector_local
     DROP COLUMN aggType,
     DROP COLUMN direction,
     DROP COLUMN podName;
+ALTER TABLE flows
```

Comment:
Should these changes be in 000005_0-6-0.down.sql or in next version like 000006_0-7-0.down.sql?

Reply:
I think these are changes part of Tushar's commit and PR, I just have them in this PR as well.

Reply:
PR has been updated, Please use the new fields and files